fasttext.py

```diff
@@ -53,6 +53,24 @@ def apply_transform(self, transform):
         transmat = np.loadtxt(transform) if isinstance(transform, str) else transform
         self.embed = np.matmul(self.embed, transmat)
 
+    def export(self, outpath):
+        """
+        Transforming a large matrix of WordVectors is expensive. 
+        This method lets you write the transformed matrix back to a file for future use
+        :param The path to the output file to be written 
+        """
+        fout = open(outpath, "w")
```

Comment:
Please also close the filehandle after you are done writing to it? (or use `with open...`)